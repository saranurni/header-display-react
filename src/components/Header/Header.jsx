import PropTypes from "prop-types";

export default function Header({ name }) {
  return <h1 className="text-3xl">Hello {name}</h1>;
}

Header.propTypes = {
  name: PropTypes.string.isRequired,
};
