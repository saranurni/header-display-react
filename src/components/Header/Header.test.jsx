import renderer from "react-test-renderer";
import Header from "./Header";

it("renders correctly", () => {
  const tree = renderer.create(<Header name="John Doe" />).toJSON();
  expect(tree).toMatchSnapshot();
});
