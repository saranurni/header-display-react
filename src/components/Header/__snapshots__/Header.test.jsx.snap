// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1
  className="text-3xl"
>
  Hello 
  John Doe
</h1>
`;
